### DISCUSSION TITLE
Disable unused dockers (TheHive & Cortex) in 2.3.100

### DISCUSSION THREAD

**Discussion by presianbg** [Security-Onion-Solutions/securityonion#7052] [Unsupported Versions]
Hi,

First of all thanks for the awesome 2.3.100 release! The built-in case management is fantastic! 
I've never used the bundled TheHive/Cortex and now they are even more obsolete!

For the sake of preserving precious resources and  keeping our setup as clean as possible, would someone point me in the right direction of disabling the `so-thehive` , `so-thehive-es` and `so-cortex` on the manager node ?

I guess the solution is to flip the switch on `thehive: 1` in `/opt/so/saltstack/local/pillar/minions/somanager_manager.sls`, but I'm not sure and not willing to break our setup. 

Cheers,
PY

**Comment by weslambert** [comment]
We left them enabled for the time being, but you should be able to disable them by stopping the services, and flipping the value to `0` in the pillar file.
